src/service/user.service.js

```diff
@@ -0,0 +1,65 @@
+import { ApiError } from '../exceptions/api.error.js';
+import { User } from '../modules/User.js';
+import { Console } from '../utils/Console.js';
+
+function validateEmail(value) {
+  const EMAIL_PATTERN = /^[\w.+-]+@([\w-]+\.){1,3}[\w-]{2,}$/;
+
+  if (!value) {
+    return 'Email is required';
+  }
+
+  if (!EMAIL_PATTERN.test(value)) {
+    return 'Email is not valid';
+  }
+}
+
+function validatePassword(value) {
+  if (!value) {
+    return 'Password is required';
+  }
+
+  if (value.length < 6) {
+    return 'At least 6 characters';
```

Comment:
Consider providing a more descriptive error message, such as 'Password must be at least 6 characters long.'